### PR TITLE
Implemented Category Create and Read

### PR DESCRIPTION
**Pull request by timzprof**

### CODE REVIEW

File: functions/src/routes/categories/index.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+const getCategories = require('./getCategories');
+const addCategory = require('./addCategory');
+
+const categoryRouter = express.Router();
+
+categoryRouter.get('/', verifyToken, getCategories);
+
+categoryRouter.post('/', verifyToken, addCategory);
+
+module.exports = categoryRouter;
```
Comment (daniellamarr) [inline functions/src/routes/categories/index.js]: Just add a new line here

**Review by daniellamarr — COMMENTED** [review]
Once you've pushed the change, merge it!

File: functions/src/routes/categories/index.js (hunk 1 above)
Comment (timzprof) [reply functions/src/routes/categories/index.js]: After `module.exports` right

Comment (timzprof) [reply functions/src/routes/categories/index.js]: It's a coding convention I presume

Comment (daniellamarr) [reply functions/src/routes/categories/index.js]: Yeah

**Review by daniellamarr — APPROVED** [review]

**Pull request merged**